src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeGenerationTests.cs

```diff
@@ -123,6 +123,27 @@ protected void AssertEqual(CodeObject c, string expected)
             }
         }
 
+        protected void AssertEqualPreserveLineBreaks(CodeObject c, string expected)
+        {
+            // Validate all identifiers are valid
+            CodeGenerator.ValidateIdentifiers(c);
+
+            // Generate code
+            CodeDomProvider provider = GetProvider();
+            string code = GenerateCode(c, provider);
+
+            // Make sure the code matches what we expected
+            try
+            {
+                Assert.Equal(CoalesceWhitespace(expected, true), CoalesceWhitespace(code, true));
+            }
+            catch
+            {
+                Console.WriteLine(code);
+                throw;
+            }
```

Comment:
Why is this try/catch necessary?

Reply:
It was just a carry-over from the existing ``AssertEqual`` method that I copied; I guess it was there to print the whole output on failure rather than just the string diff.  It won't print reliably anyway in parallel test execution anyway, so I should take it out. 

Should I remove it from the existing ``AssertEqual`` method?

Reply:
After mulling over this I decided to remove it from the existing method as well; just because it's old code doesn't make it right, plus I'm in the area.